internal/commands/app/create_test.go

```diff
@@ -335,75 +330,59 @@ Check out your app: cd ./test-app && realm-cli app describe
 			procedure(console)
 		}()
 
-		var createdApp realm.App
-		rc := mock.RealmClient{}
-
-		zipPkg, err := zip.OpenReader("testdata/project.zip")
-		assert.Nil(t, err)
-		defer zipPkg.Close()
-
-		rc.ExportFn = func(groupID, appID string, req realm.ExportRequest) (string, *zip.Reader, error) {
-			return "", &zipPkg.Reader, err
-		}
-		rc.CreateAppFn = func(groupID, name string, meta realm.AppMeta) (realm.App, error) {
-			createdApp = realm.App{
-				GroupID:     groupID,
-				ID:          "456",
-				ClientAppID: name + "-abcde",
-				Name:        name,
-				AppMeta:     meta,
-			}
-			return createdApp, nil
-		}
-		rc.ImportFn = func(groupID, appID string, appData interface{}) error {
-			return nil
-		}
-		rc.AllTemplatesFn = func() ([]realm.Template, error) {
-			return []realm.Template{
-				{
-					ID:   "palm-pilot.bitcoin-miner",
-					Name: "Mine bitcoin on your Palm Pilot from the comfort of your home, electricity not included",
-				},
-				{
-					ID:   "blackberry.important-business-app",
-					Name: "Oh wow, a Blackberry... you must a very powerful, extravagant man.",
-				},
-			}, nil
+		testApp := realm.App{
+			ID:          "456",
+			GroupID:     "123",
+			ClientAppID: "bitcoin-miner-abcde",
+			Name:        "bitcoin-miner",
 		}
 
-		clientZipPkg, err := zip.OpenReader("testdata/react-native.zip")
+		backendZipPkg, err := zip.OpenReader("testdata/project.zip")
 		assert.Nil(t, err)
-		rc.ClientTemplateFn = func(groupID, appID, templateID string) (*zip.Reader, bool, error) {
-			return &clientZipPkg.Reader, true, nil
-		}
-		ac := mock.AtlasClient{}
-		ac.GroupsFn = func() ([]atlas.Group, error) {
-			return []atlas.Group{{ID: "123"}}, nil
-		}
-		ac.ClustersFn = func(groupID string) ([]atlas.Cluster, error) {
-			return []atlas.Cluster{{Name: "test-cluster"}}, nil
+
+		client := mock.RealmClient{
```

Comment:
[nit] can we rename this to `realmClient` and define `atlasClient` similarly alongside here (rather than inline to the `cmd.Handler` call below)?